apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -171,16 +211,39 @@ export default class EnrollmentsPanel extends React.Component {
           <Button
             bsSize="xsmall"
             disabled={this.state.selectedEnrollments.length === 0}
-            onClick={this.handleClickMove}
+            onClick={this.handleClickChangeEnrollments}
+            name={BUTTON_NAMES.moveEnrollment}
           >
             Move (admin)
             <MoveEnrollmentsDialog
-              show={this.state.isMoveEnrollmentsDialogOpen}
+              show={
+                this.state.enrollmentChangeDialogOpen ===
+                BUTTON_NAMES.moveEnrollment
+              }
               selectedEnrollments={this.state.selectedEnrollments}
-              onCancel={this.handleMoveEnrollmentsCanceled}
+              onCancel={this.handleChangeEnrollmentsCanceled}
               onMove={this.handleMoveEnrollmentsConfirmed}
             />
           </Button>
+        )}{' '}
```

Comment:
Kinda ugly, but does the trick!

Reply:
And very common!
![image](https://user-images.githubusercontent.com/1615761/74861526-5a19cd80-52ff-11ea-80e9-3cffbe92c0ed.png)

Reply:
@molly-moen it just puts a space between the "Move (admin)" and "Edit name (admin)" buttons 😃